### ISSUE TITLE
Support for PureOS, a debian flavour linux distro

### ISSUE THREAD

**Issue by aurovrata** [nodesource/distributions#574]
```
$ curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
## Installing the NodeSource Node.js v8.x repo...
## Populating apt-get cache...
## Confirming "green" is supported...
+ curl -sLf -o /dev/null 'https://deb.nodesource.com/node_8.x/dists/green/Release'
## Your distribution, identified as "green", is not currently supported, please contact NodeSource at https://github.com/nodesource/distributions/issues if you think this is incorrect or would like your distribution to be considered for support
```

**Comment by sneurlax** [comment]
Please yes, PureOS is "just" Ubuntu 16.04 ... "green" == "xenial"

**Comment by sneurlax** [comment]
See https://github.com/nodesource/distributions/pull/602, I have tested 9.x.  This might work for you:

```
curl -sL https://raw.githubusercontent.com/sneurlax/distributions/8531cf52c1c0cc723d03cdb28d617eb6550bddfc/deb/setup_8.x | sudo -E bash -
sudo apt-get install -y nodejs
```

(I would not recommend that you do this because of the obvious security risk of running someone else's script, but simply edit `setup_8.x` near line 280 to say `check_alt "PureOS" "green" "Ubuntu" "xenial"`, run it (you may need to `sudo chmod +x ./setup_8.x` first,) and `sudo apt-get install nodejs`)

**Comment by aurovrata** [comment]
yes, thanks.  I also used Debian 9 stretch in my sources, and got it to install and work.  I can't recall why I needed it, but I think it was to get an apache kafka server to run on my Librem.

**Comment by sneurlax** [comment]
I just got set up on a Librem for regular everyday work and encountered the issue that way.

Would it be more correct that PureOS is Debian 9 stretch, not Ubuntu 16.04 specifically?  Maybe I need to research PureOS a bit more...

**Comment by aurovrata** [comment]
that's a good question, the PureOS.net page clearly says it is a fork of Debian, but is it 8 or 9 isn't so clear.

**Comment by chrislea** [comment]
Closed by 3070143.
